statsmodels/stats/sandwich_covariance.py

```diff
@@ -246,7 +246,7 @@ def _get_sandwich_arrays(results):
             hessian_inv = np.asarray(results.normalized_cov_params)
 
         # experimental support for freq_weights
-        if hasattr(results.model, 'freq_weights'):
+        if hasattr(results.model, 'freq_weights') and not cov_type == 'clu':
```

Comment:
The first condition might now be empty for GLM because it's currently always true for GLM. But it will be required for other models.